src/04-modality-specific-files/03-electroencephalography.md

```diff
@@ -20,7 +20,10 @@ stored in one of the following formats:
 
 -   [European data format](https://www.edfplus.info/)
     (including [`edf+`](https://www.edfplus.info/specs/edfplus.html);
-    each recording consisting of a `.edf` file)
+    each recording consisting of a `.edf` file). Although the
+    European data format supports
+    both upper and lower case extensions, in BIDS the file extension MUST
+    be lower case (that is `.edf` instead of `.EDF`).
```

Comment:
I'm okay with this as-is, but I feel like it's over-explaining. We can simply say:

```suggestion
    each recording consisting of a `.edf` file).
    The capital `.EDF` extension MUST NOT be used.
```

Is it worth taking the opportunity to convert extensions to a table?

| Format | Extension(s) | Description |
|--|--|--|
| [European data format](https://www.edfplus.info/) | `.edf` | Each recording consists of a single file. [`edf+`](https://www.edfplus.info/specs/edfplus.html) files are permitted. The capital `.EDF` extension MUST NOT be used. |
| ... |

Reply:
Converted to table.